apps/i18n/codebridge/en_us.json

```diff
@@ -1,4 +1,5 @@
 {
+  "addToAiTutorContext": "Add to AI Tutor Chat",
```

Comment:
Do we still need to add loc strings?

Reply:
Not sure, but I'm going to keep doing it until there's an official announcement 😄